src/common/components/giftViews/giftListItem/giftListItem.component.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import template from './giftListItem.tpl';
+import desigSrc from 'common/directives/desigSrc.directive';
+
+let componentName = 'giftListItem';
+
+class GiftListItemController {
+
+  /* @ngInject */
+  constructor() {
+  }
+}
+
+export default angular
+  .module( componentName, [
+    template.name,
+    desigSrc.name
+  ] )
+  .component( componentName, {
+    controller:  GiftListItemController,
+    templateUrl: template.name,
+    transclude:  {
+      'selectInput': '?label'
+    },
+    bindings:    {
+      gift:        '<',
```

Comment:
[Right here](https://github.com/CruGlobal/give-web/blob/stop-gift-step-1/src/app/profile/yourGiving/stopStartRecurringGifts/stopGift/stopGiftStep1.component.js#L18). This function is called by the stopGiftStep1 "Continue" button and is passing back to the  parent stopGift component the selected gifts from the list. It's somewhat open how the developer or component wants to get the selected gifts back. The gift objects are modified in place, so a component could just call next() and the parent can find selected ones by filtering.